test/solidity/Facets/SymbiosisFacet.t.sol

```diff
@@ -0,0 +1,153 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity 0.8.17;
+
+import { LibAllowList, TestBaseFacet, console } from "../utils/TestBaseFacet.sol";
+import { TestFacet } from "../utils/TestBase.sol";
+import { SymbiosisFacet } from "lifi/Facets/SymbiosisFacet.sol";
+import { ISymbiosisMetaRouter } from "lifi/Interfaces/ISymbiosisMetaRouter.sol";
+
+// Stub SymbiosisFacet Contract
+contract TestSymbiosisFacet is SymbiosisFacet, TestFacet {
+    constructor(
+        ISymbiosisMetaRouter _symbiosisMetaRouter,
+        address _symbiosisGateway
+    ) SymbiosisFacet(_symbiosisMetaRouter, _symbiosisGateway) {}
+}
+
+contract MockMetaRouter {
+    address internal constant SYMBIOSIS_GATEWAY =
+        0x25bEE8C21D1d0ec2852302fd7E674196EA298eC6;
+
+    function metaRoute(
+        ISymbiosisMetaRouter.MetaRouteTransaction calldata mt
+    ) external payable {
+        if (!mt.nativeIn) {
+            bytes memory callData = abi.encodeWithSignature(
```

Comment:
A comment/description why we are using this contract here would be helpful. Like why are you not deploying the facet with the actual meta router? How do we make sure that the facet works with the actual meta router?